src/transport/CASESession.h

```diff
@@ -147,6 +210,9 @@ class DLL_EXPORT CASESession
     void SendErrorMsg(SigmaErrorType errorCode);
     void HandleErrorMsg(const PacketHeader & header, const System::PacketBufferHandle & msg);
 
+    // TODO
```

Comment:
TODO?

Reply:
Do we have such mechanism in CHIP repo? To obtain current date and time?